providers/aws-ec2/src/main/java/org/jclouds/aws/ec2/features/RouteTableApi.java

```diff
@@ -0,0 +1,277 @@
+import org.jclouds.rest.annotations.RequestFilters;
+import org.jclouds.rest.annotations.VirtualHost;
+import org.jclouds.rest.annotations.XMLResponseParser;
+
+import com.google.common.collect.FluentIterable;
+
+/**
+ * Provides access to AWS Route Table services.
+ *
+ * @see <a href="http://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_RouteTable.html">RouteTable docs</a>
+ */
+@RequestFilters(FormSigner.class)
+@VirtualHost
+@Path("/")
+public interface RouteTableApi {
+
+   /**
+    * Creates a {@link RouteTable}
+    *
+    * @param region The region to create the table in.
+    * @param vpcId The ID of the VPC
+    * @return The route table
+    */
+   @Named("CreateRouteTable")
+   @POST
+   @FormParams(keys = ACTION, values = "CreateRouteTable")
+   @XMLResponseParser(CreateRouteTableResponseHandler.class)
+   RouteTable createRouteTable(
+      @EndpointParam(parser = RegionToEndpointOrProviderIfNull.class) @Nullable String region,
+      @FormParam("VpcId") String vpcId);
+
+   /**
+    * Creates a {@link RouteTable}, supplying options.
+    *
+    * @param region  The region to create the table in
+    * @param vpcId The ID of the VPC
+    * @param options Options for the request
+    * @return The route table
+    */
+   @Named("CreateRouteTable")
+   @POST
+   @FormParams(keys = ACTION, values = "CreateRouteTable")
+   @XMLResponseParser(CreateRouteTableResponseHandler.class)
+   RouteTable createRouteTable(
+      @EndpointParam(parser = RegionToEndpointOrProviderIfNull.class) @Nullable String region,
+      @FormParam("VpcId") String vpcId,
+      RouteTableOptions options);
+
+   /**
+    * Deletes a {@link RouteTable}
+    *
+    * @param region The region to delete the table from
+    * @param routeTableId The ID of the table to delete
+    * @return true if the route table was found and deleted
+    */
+   @Named("DeleteRouteTable")
+   @POST
+   @FormParams(keys = ACTION, values = "DeleteRouteTable")
+   @XMLResponseParser(ReturnValueHandler.class)
+   @Fallback(Fallbacks.FalseOnNotFoundOr404.class)
+   boolean deleteRouteTable(
+      @EndpointParam(parser = RegionToEndpointOrProviderIfNull.class) @Nullable String region,
+      @FormParam("RouteTableId") String routeTableId);
+
+   /**
+    * Delete a {@link RouteTable}, supplying options.
+    *
+    * @param region  The region to delete the table from
+    * @param routeTableId The ID of the table to delete
+    * @param options Options for the request
+    * @return true if the route table was found and deleted
+    */
+   @Named("DeleteRouteTable")
+   @POST
+   @FormParams(keys = ACTION, values = "DeleteRouteTable")
+   @XMLResponseParser(ReturnValueHandler.class)
+   @Fallback(Fallbacks.FalseOnNotFoundOr404.class)
+   boolean deleteRouteTable(
+      @EndpointParam(parser = RegionToEndpointOrProviderIfNull.class) @Nullable String region,
+      @FormParam("RouteTableId") String routeTableId,
+      RouteTableOptions options);
```

Comment:
This request actually only supports the `dryRun` parameter. Worth changing the signature to just accept a boolean.